agent/wptdriver/wpt_test.h

```diff
@@ -225,6 +225,10 @@ class WptTest {
   void      BuildScript();
   CAtlList<ScriptCommand> _script_commands;
 
+  //Timeout information
+  DWORD    _overall_timeout_in_ms;
+  bool     _has_test_timed_out;
+

```

Comment:
can you move this just after _test_timeout declaration? the section where you added it right now has a comment on top: // system information but the fields you are adding has to do with test configuration.